### ISSUE TITLE
Question: Comparison w/ similar projects

### ISSUE THREAD

**Issue by brandonmp** [tshaddix/react-chrome-redux#6]
Thanks for this project & the video presentation you did. I'm new to chrome extensions, so your level of documentation/explanation is great--def. best I've found on a project like this.

But on that note I was wondering: can you talk a bit about your motivation for creating this project vice using one of the small # of similar projects? e.g. [crossbuilder](https://github.com/zalmoxisus/crossbuilder) w/ 'cross-messaging' ?

Asking b/c as a newbie, it's tough to differentiate & wanna make sure i understand which tools fit which projects.

**Comment by tshaddix** [comment]
Hey @brandonmp ! I'm glad the video is helpful! I'm hoping to get some better docs up in the near future (#5) .

I have never used `crossbuilder`, but it looks like a really cool project!

I built this project to be a utility more than any sort of build/project system. My main goal was to take as much as a hands-off approach as possible, and let developers still write chrome extensions how they are used to. It looks like `crossbuilder` is much more oriented towards abstracting the complexities of each of the platform away, which is very impressive. This abstraction comes with the need to include a few dependencies as well as a somewhat strict project structure. 

It really comes down to what you're looking for! We've found `react-chrome-redux` project very useful at [GoGuardian](https://goguardian.com) because we only write Chrome Extensions (vs other browsers/platforms) and we wanted to be able to add on and extend our existing code without needing to rewrite the entire extension. 

TL;DR; This project is a utility library for "enabling" React+Redux while working in Chrome Extensions while `crossbuilder` is a full on build/project system for building cross-platform apps.

Once again, I've never used `crossbuilder`, so my thoughts are completely based off of their README. I hope this helps!
